code/lib/test/src/index.ts

```diff
@@ -84,12 +94,24 @@ export const traverseArgs = (value: unknown, depth = 0, key?: string): unknown =
   return value;
 };
 
-const nameSpiesAndWrapActionsInSpies: LoaderFunction<Renderer> = ({ initialArgs }) => {
+const nameSpiesAndWrapActionsInSpies: LoaderFunction = ({ initialArgs }) => {
   traverseArgs(initialArgs);
 };
 
+const enhanceContext: LoaderFunction = (context) => {
+  if (globalThis.HTMLElement && context.canvasElement instanceof globalThis.HTMLElement) {
+    context.canvas = within(context.canvasElement);
+    // TODO enable this in a later PR, once we have time to QA this properly
+    // context.userEvent = userEvent.setup();
+  }
+};
+
 // We are using this as a default Storybook loader, when the test package is used. This avoids the need for optional peer dependency workarounds.
 // eslint-disable-next-line no-underscore-dangle
-(global as any).__STORYBOOK_TEST_LOADERS__ = [resetAllMocksLoader, nameSpiesAndWrapActionsInSpies];
+(global as any).__STORYBOOK_TEST_LOADERS__ = [
```

Comment:
Will this be loaded in the right moment in portable stories? Else we might need to turn this into a preset as part of an addon?

Reply:
This will be working correctly because it only gets executed in applyLoaders so there is more than enough time to make PS work